test/Crowdsale.js

```diff
@@ -19,11 +22,16 @@ contract('Crowdsale', function ([_, investor, wallet, purchaser]) {
 
   const expectedTokenAmount = rate.mul(value)
 
+  before(async function() {
+    //Advance to the next block to correctly read time in the solidity "now" function interpreted by testrpc
+    await advanceBlock()
```

Comment:
I don't understand why this is in a `before` block instead of `beforeEach`. Could you clarify?